client/src/main/java/org/apache/rocketmq/client/impl/MQClientAPIImpl.java

```diff
@@ -2007,6 +2007,7 @@ public TopicRouteData getTopicRouteInfoFromNameServer(final String topic, final
             case ResponseCode.TOPIC_NOT_EXIST: {
                 if (allowTopicNotExist) {
                     log.warn("get Topic [{}] RouteInfoFromNameServer is not exist value", topic);
+                    return null;
```

Comment:
This may cause sime npe, maybe need to check, such as https://github.com/apache/rocketmq/blob/ad027560780c2af801e4bcb447d85bf0df62071a/proxy/src/main/java/org/apache/rocketmq/proxy/service/admin/DefaultAdminService.java#L60-L85